BDArmory/Distribution/GameData/BDArmory/Parts/decoyDropper/Bubbler.cfg

```diff
@@ -0,0 +1,76 @@
+RESOURCE_DEFINITION
```

Comment:
Shouldn't the RESOURCE_DEFINITIONs for countermeasures be in the Resources/Countermeasures.cfg file?
Similarly for CMDecoy in Spoofer.cfg?

Reply:
Probably, yes. I'll move them over.